### PR TITLE
Add Stefan Marr's ReBench suite

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2797,3 +2797,13 @@ @misc{v8
     year = {2008},
     author = {Google},
 }
+
+@misc{rebench18marr,
```
Comment (ltratt) [inline softdev.bib:2801]: I think this file is mostly 4-space indentation? If so, can we match that?

Comment (jacob-hughes) [reply softdev.bib:2801]: Sure,  fixed: f3963a892cbd2177a05d223e6f41a0f0461c6914

**Comment by ltratt** [comment]
Please squash.

**Comment by jacob-hughes** [comment]
Squashed

**Pull request merged**